charts/core/src/Chart/Chart.styles.ts

```diff
@@ -1,22 +1,16 @@
 import { css } from '@leafygreen-ui/emotion';
-import { Theme } from '@leafygreen-ui/lib';
-import {
-  borderRadius,
-  color,
-  InteractionState,
-  Variant,
-} from '@leafygreen-ui/tokens';
 
-export const chartStyles = css`
-  grid-row: 2;
+export const chartContainerStyles = css`
+  display: grid;
+  grid-template-areas:
+    'chartHeader'
```

Comment:
nit: Do you explicitly state anywhere that the grid-area for the `headerContent` is `chartHeader`?

Reply:
I'm not sure that's correct. `chartHeader` is the grid-area for `Header`, `headerContent` is a node that gets ultimately inside of the `Header`. Am I misunderstanding what's being asked here?

Reply:
My bad! I confused myself. Where did you add `grid-area: chartHeader`?

Reply:
Great catch! Totally missed it. Since I happened to order it correctly in storybook I didn't realize I didn't actually add the area to the component. Will update!